api/handlers.go

```diff
@@ -469,7 +520,7 @@ func (api *API) userPUT(w http.ResponseWriter, req *http.Request, _ httprouter.P
 		api.WriteError(w, err, http.StatusBadRequest)
 		return
 	}
-	var payload userUpdateData
+	var payload userUpdateDTO
 	err = json.Unmarshal(bodyBytes, &payload)
```

Comment:
We should make the `UnmarshalPayload` method so we can use it here since we aren't normalizing it immediately which probably should be updated.